content/_data/authors/alexsomai.adoc

```diff
@@ -0,0 +1,5 @@
+---
+name: "Alexandru Somai"
+twitter: alex_somai
+github: alexsomai
```

Comment:
Not important (NIT): Would be great if you add some details to your GitHub account.